src/sections/offering/get-insured.tsx

```diff
@@ -0,0 +1,60 @@
+import { colors } from '@hedviginsurance/brand'
+import {
+  GetInsuredButton,
+  LinkTag,
+} from 'components/offering/get-insured-button'
+import { Header } from 'components/offering/header'
+import { InnerWrapper } from 'components/offering/inner-wrapper'
+import { Wrapper } from 'components/offering/wrapper'
+import * as React from 'react'
+import styled from 'react-emotion'
+import * as VisibilitySensor from 'react-visibility-sensor'
+
+interface Props {
+  title: string
+  name: string
+  getInsured: string
```

Comment:
i think this could easily be confused with the more traditional java `getter` (`getXX` for getting arbitrary value), maybe we could find a better name?